src/dotnet-install.ps1

```diff
@@ -434,6 +437,51 @@ function Get-LegacyDownload-Link([string]$AzureFeed, [string]$SpecificVersion, [
     return $PayloadURL
 }
 
+function Get-Product-Version([string]$AzureFeed, [string]$SpecificVersion) {
```

Comment:
Please consider making $Runtime one of the parameters instead of inheriting it from parent scope.

Reply:
Let's abandon this idea, this function is three or so levels deep and passing around $Runtime would be just an unnecessary change.